requirements.txt

```diff
@@ -2,19 +2,19 @@ numpy==1.18.1
 pandas
 seaborn
 matplotlib
-numba<=0.46
-scanpy==1.4.6
-anndata==0.7.1
+numba
+scanpy
+anndata
 h5py<3
 rpy2==3.1.0
 anndata2ri==1.0.2
 scipy
 scikit-learn
 scikit-misc
 loompy==3.0.6
-louvain==0.6.1
-umap-learn==0.3.10
+louvain
+umap-learn
```

Comment:
I wonder about these 2. There is a louvain issue with >0.6.1 if I recall, and `umap-learn` should maybe be <0.5.0 if possible.